test/support/rails_application_fixture_helper.rb

```diff
@@ -0,0 +1,75 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require "rails_test_helper"
+
+module RailsApplicationFixtureHelper
+  include ApplicationFixtureHelper
+
+  def setup_application_fixture
+    super()
```

Comment:
Have ApplicationFixtureHelper set up and tear down the temporary application fixture, then manage the Rails state.

Reply:
Should we maybe rename the method to `setup_rails_application_fixture` to be more explicit about what is happening? There are benefits of allowing them to have the same name and using separate method names. I don't have a strong preference, but I would like to know how you think about it.

Reply:
I also prefer to have ApplicationFixtureHelper and RailsApplicationFixtureHelper present the same interface to test authors.

Reply:
> I also prefer to have ApplicationFixtureHelper and RailsApplicationFixtureHelper present the same interface to test authors.

I figured because you implemented it this way. What I want to understand is why you think this is the correct way and when you think it makes sense to have two distinct names?

Reply:
The setup and teardown methods are part of this helper class' public interface.  Public methods names should aim to be declarative and short.  They should not explain _how_ something will be accomplished (something that is permissible in private helper methods), nor list all of the side effects, nor indicate where the method is defined.

~~`ApplicationFixtureHelper#copy_temp_directory_and_set_cwd`~~ --> `ApplicationFixtureHelper#setup_application_fixure`

~~`RailsApplicationFixtureHelper#copy_temp_directory_and_set_cwd_and_cache_rails_paths`~~ --> `RailsApplicationFixtureHelper#setup_application_fixure`

I also chose to override `#setup_application_fixture` so testers would not have access to both a `#setup_application_fixture` and a ``#setup_rails_application_fixture` at the same time.  Having both definitions provides no additional functionality, and only gives test implementers more ways to do something bad.